ttter/accounts/forms.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+from .models import MyUser
```

Comment:
```suggestion
MyUser = get_user_model()
```
で統一しましょう。
`from .models  import MyUser`でもMyUserモデルは取ってこれるのですが、get_user_model()で取ってくる方向でいきましょう。